.github/no-response.yml

```diff
@@ -0,0 +1,13 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 10
+# Label requiring a response
+responseRequiredLabel: "STATE: Need clarification"
```

Comment:
'more-information-needed' in our case

Reply:
I'll rename this label to "STATE: Need clarification". Unfortunatly this label's name is hard coded in the robot ((